### ISSUE TITLE
Combine all stylesheets

### ISSUE THREAD

**Issue by cmb69** [cmsimple-xh/cmsimple-xh#140]
[Forum discussion](http://www.cmsimpleforum.com/viewtopic.php?f=5&t=10540&start=10#p52883)

**Comment by TN03** [comment]
Combining all stylesheets to one file might be a good idea. But moving the template to the end could cause problems with @import rules in the template-css, which have to be at top of the css file AFAIK

**Comment by cmb69** [comment]
> But moving the template to the end could cause problems with @import rules in the template-css, which have to be at top of the css file AFAIK

Good catch – thanks! Indeed, @import at-rules would cause issues, but we could work around that by excluding stylesheets with such rules, or moving the rules to the top. However, I'd probably prefer to combine only core.css and plugins.css which is supposed to be easily done, and would eschew issues with multiple templates on the site.

**Comment by cmb69** [comment]
I have documented that [plugin stylesheets shouldn't use `@import` at-rules](http://www.cmsimple-xh.org/wiki/doku.php/developers_manual?do=diff&rev2%5B0%5D=1491215334&rev2%5B1%5D=1491952343&difftype=sidebyside).

**Comment by cmb69** [comment]
I suggest to merge PR #231 (which ignores the template stylesheets to avoid the mentioned issues).

**Comment by TN03** [comment]
Hmm, ich bin jetzt etwas durcheinander.
Mit https://github.com/cmsimple-xh/cmsimple-xh/issues/141 haben wir ja das Template in der Ladereihenfolge nach hinten gesetzt.
Wäre es dann nicht konsequent es auch nach die Ausgabe von $hjs zu schieben?
Der Sinn war ja, dass im Template Plugin-Styles überschrieben werden können. Viele Plugins laden die relevanten Styles für die Ausgabe aber aus separaten Dateien über $hjs. Ich rede hier von Assets wie FancyBox, ColorBox, FotoRama usw. Um hier Anpassungen zu machen ist im Template nach wie vor "!important" nötig.

**Comment by cmb69** [comment]
> Wäre es dann nicht konsequent es auch nach die Ausgabe von $hjs zu schieben?

Ja, ich denke schon. Brauchen wir dafür eine extra Abstimmung?

**Comment by TN03** [comment]
> Brauchen wir dafür eine extra Abstimmung?

Von mir aus nicht.
